odk1-src/collect-import-export.rst

```diff
@@ -1,50 +1,43 @@
-Importing and Exporting Settings
-====================================
+Configuring Collect via QR code
+================================
 
 .. image:: /img/collect-settings/import-settings.*
   :alt: Import/export settings menu of ODK Collect
   :class: device-screen-vertical
 
-QR Code
-----------
-
-QR Code or the Quick Response Code is a two dimensional barcode. QR codes can be used to configure Collect on many devices. The QR image presented contains all of your current General and Admin settings, including admin and server passwords. When this QR code is scanned from the ODK Collect app on another device, these settings are imported.
-
-Sharing QR code
+Settings QR codes
 ------------------
 
-You can click on the |share| icon to share the QR code as an image. When you click on it, it displays a list of applications and services like *whatsapp*, *facebook*, *hangouts*, *bluetooth*, *MMS* to name a few, which can be used to share the QR code. This is useful when there are several different data collection sites and all devices have to be configured in the same way, in which case the QR code can be shared from one reference device. 
+QR codes are a type of barcode that can encode a lot of data with low risk of error. These can be used to configure Collect on a fleet of devices. The QR image displayed on the :guilabel:`Configure via QR code` screen contains all of your current General and Admin settings. When this QR code is scanned from the ODK Collect app on another device, the settings from the code are imported.
```

Comment:
Not sure we need a description of what a QR code is here? Maybe you put that in based on people being confused.

Reply:
Ah, good point. I was just blindly improving what was there before.